src/rviz/default_plugin/odometry_display.cpp

```diff
@@ -383,18 +383,30 @@ void OdometryDisplay::update(float /*wall_dt*/, float /*ros_dt*/)
   if (!last_used_message_ || !ref_pose_)
     return;
 
+  std::string frame = last_used_message_->header.frame_id;
```

Comment:
Please use `const std::string&`. No need to copy the string.

Reply:
I dropped this commit anyway as this part is already handled by MessageFilterDisplay